### DISCUSSION TITLE
How to log build output to a file?

### DISCUSSION THREAD

**Discussion by RolfNoot** [microsoft/vscode-cmake-tools#2866] [Q&A] [answered]
How can the build output be saved to a file and still have it in the output window? Preferably using a 'user scope' setting.

**Comment by bobbrow** [comment, marked as answer]
We don't have this feature in the extension today.  But we do have a comprehensive log that is saved in your operating system's data folder.
* `%LocalAppData%\CMakeTools\log.txt` on Windows
* `$XDG_DATA_HOME/CMakeTools/log.txt` on linux/mac if that variable is defined
* `~/.local/share/CMakeTools/log.txt` otherwise

**Reply by RolfNoot** [reply]
Thank you. Unfortunately that log is adding lots of information which doesn't reflect the latest project build output only.
